test cases/common/38 string operations/meson.build

```diff
@@ -101,3 +101,11 @@ assert('\\\\n' == bs_bs_n, 'Four backslash broken before n')
 assert('\\\\\n' == bs_bs_nl, 'Five backslash broken before n')
 assert('\\\\' == bs_bs, 'Double-backslash broken')
 assert('\\' == bs, 'Backslash broken')
+
+mysubstring='foobarbaz'
+assert(mysubstring.substring() == 'foobarbaz', 'substring is broken')
+assert(mysubstring.substring(0) == 'foobarbaz', 'substring is broken')
+assert(mysubstring.substring(1) == 'oobarbaz', 'substring is broken')
+assert(mysubstring.substring(-5) == 'foobarbaz', 'substring is broken')
+assert(mysubstring.substring(1, 4) == 'oob', 'substring is broken')
+assert(mysubstring.substring(1,-5) == 'oobarbaz', 'substring is broken')
```

Comment:
I would expect an `end` value of `-5` to mean `5 characters from the end`, like python would do for `mysubstring[1:-5]` (which would return `oob` here). Otherwise it should just be a hard error imho.

Reply:
done